job-service-postgres-container/pom.xml

```diff
@@ -147,7 +147,7 @@
                             <alias>job-service-postgres</alias>
                             <name>${dockerJobServiceOrg}job-service-postgres${dockerProjectVersion}</name>
                             <build>
-                                <from>${dockerHubPublic}/cafapi/prereleases:java-postgres-3.0.0-SNAPSHOT</from>
+                                <from>${dockerHubPublic}/cafapi/java-postgres:3.0.0</from>
```

Comment:
@vishalkumar-barnwal @frankmccarry Docker images should be 1 part numbers, in this case:
```
<from>${dockerHubPublic}/cafapi/java-postgres:3</from>
```